### ISSUE TITLE
[Breadcrumb][A11y]: Focus indicator for 'More' icon fully covers the control when zoomed in

### ISSUE THREAD

**Issue by Brahmansh** [SAP/ui5-webcomponents#11336]
### Bug Description

When the page is zoomed to 200%, the 'More' icon control is not visible upon receiving keyboard focus, as the focus indicator fully covers the control, rendering the icon unrecognizable to users.

### Affected Component

Breadcrumb

### Expected Behaviour

 The 'More' icon control should remain fully visible and clearly identifiable when focused via keyboard, without being obscured by the focus indicator.

### Isolated Example

https://sap.github.io/ui5-webcomponents/play/#eyJpbmRleC5odG1sIjp7Im5hbWUiOiJpbmRleC5odG1sIiwiY29udGVudCI6IjwhLS0gcGxheWdyb3VuZC1mb2xkIC0tPlxuPCFET0NUWVBFIGh0bWw-XG48aHRtbCBsYW5nPVwiZW5cIj5cblxuXG48aGVhZD5cbiAgICBcbiAgICA8c3R5bGU-XG4gICAgICAqOm5vdCg6ZGVmaW5lZCkge1xuICAgICAgICBkaXNwbGF5OiBub25lO1xuICAgICAgfVxuXG4gICAgaHRtbCB7XG4gICAgICBmb3JjZWQtY29sb3ItYWRqdXN0OiBub25lO1xuICAgIH1cbiAgICA8L3N0eWxlPlxuXG4gICAgPG1ldGEgY2hhcnNldD1cIlVURi04XCI-XG4gICAgPG1ldGEgbmFtZT1cInZpZXdwb3J0XCIgY29udGVudD1cIndpZHRoPWRldmljZS13aWR0aCwgaW5pdGlhbC1zY2FsZT0xLjBcIj5cbiAgICA8dGl0bGU-U2FtcGxlPC90aXRsZT5cbjwvaGVhZD5cblxuPGJvZHkgc3R5bGU9XCJiYWNrZ3JvdW5kLWNvbG9yOiB2YXIoLS1zYXBCYWNrZ3JvdW5kQ29sb3IpXCI-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQtZW5kIC0tPlxuXG4gICAgPHVpNS1icmVhZGNydW1icz5cbiAgICAgICAgPHVpNS1icmVhZGNydW1icy1pdGVtIGhyZWY9XCJodHRwczovL3d3dy5zYXAuY29tXCIgdGFyZ2V0PVwiX2JsYW5rXCI-Um9vdCBQYWdlIDwvdWk1LWJyZWFkY3J1bWJzLWl0ZW0-XG4gICAgICAgIDx1aTUtYnJlYWRjcnVtYnMtaXRlbSBocmVmPVwiaHR0cHM6Ly93d3cuc2FwLmNvbVwiIHRhcmdldD1cIl9ibGFua1wiPlBhcmVudCBQYWdlPC91aTUtYnJlYWRjcnVtYnMtaXRlbT5cbiAgICAgICAgPHVpNS1icmVhZGNydW1icy1pdGVtPkN1cnJlbnQgUGFnZTwvdWk1LWJyZWFkY3J1bWJzLWl0ZW0-XG4gICAgPC91aTUtYnJlYWRjcnVtYnM-XG4gICAgPCEtLSBwbGF5Z3JvdW5kLWZvbGQgLS0-XG4gICAgPHNjcmlwdCB0eXBlPVwibW9kdWxlXCIgc3JjPVwibWFpbi5qc1wiPjwvc2NyaXB0PlxuPC9ib2R5PlxuXG48L2h0bWw-XG48IS0tIHBsYXlncm91bmQtZm9sZC1lbmQgLS0-XG4ifSwibWFpbi5qcyI6eyJuYW1lIjoibWFpbi5qcyIsImNvbnRlbnQiOiIvKiBwbGF5Z3JvdW5kLWhpZGUgKi9cbmltcG9ydCBcIi4vcGxheWdyb3VuZC1zdXBwb3J0LmpzXCI7XG4vKiBwbGF5Z3JvdW5kLWhpZGUtZW5kICovXG5pbXBvcnQgXCJAdWk1L3dlYmNvbXBvbmVudHMvZGlzdC9CcmVhZGNydW1icy5qc1wiO1xuaW1wb3J0IFwiQHVpNS93ZWJjb21wb25lbnRzL2Rpc3QvQnJlYWRjcnVtYnNJdGVtLmpzXCI7XG4ifX0

### Steps to Reproduce

1. Go to above link to the breadcrumb playground
2. Reduce window size until 'More' icon is rendered
3. Focus on more icon using keyboard and observe that focus indicator completely covers the icon.

### Log Output, Stack Trace or Screenshots

<img width="1722" alt="Image" src="https://github.com/user-attachments/assets/65a4be1c-9dec-4302-9555-8bf4b31e6832" />

### Priority

None

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Operating System

MacOS Sequoia 15.4

### Organization

SF

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Valkanister** [comment]
Hello @SAP/ui5-webcomponents-topic-p ,
Please have a look at this issue with the focus indication of the Breadcrumbs. 

Kind regards, 

Georgi

SAP Development Support
